### ISSUE TITLE
'average' aggregation type is incorrectly calculating values

### ISSUE THREAD

**Issue by nothing-personal-gh** [amcharts/amcharts5#652]
**Bug description**

When we use `average` for `valueYGrouped` in a `LineSeries`, the plotted values are incorrect. If we use `low` or `high`, it's fine.

Use this codepen for reproducing this bug - https://codepen.io/trooper-isa/pen/XWqqQOR?editors=0010 (line 61)

Consider the following data from the chart:

```
    {
      "value": 8.35,
      "time": 1664719200000 // 2022-10-03 01:00:00 GMT+11:00
    },
    {
      "value": 8.31,
      "time": 1664719500000
    },
    {
      "value": 8.35,
      "time": 1664719800000
    },
    {
      "value": 8.25,
      "time": 1664720100000
    },
    {
      "value": 8.28,
      "time": 1664720400000
    },
    {
      "value": 7.74,
      "time": 1664720700000
    }
```

The 'average' value for 01:00 period should be `(8.35 + 8.31 + 8.35)/3 = 8.33` but it is `16.20`. Set `valueYGrouped` to either `low` or `high` and see the values getting calculated correctly. 

<img width="1393" alt="Screenshot 2022-10-03 at 12 44 23 pm" src="https://user-images.githubusercontent.com/111722528/193717296-bd9afcf4-b098-4360-90ac-136a19239668.png">

 - amCharts version - 5 latest
 - Browser and its version - Chrome latest
 - Related frameworks and their versions [e.g. Angular, TypeScript, etc.] - N/A

**Comment by zeroin** [comment]
Thanks, I am looking into it. When this wrong value is shown, which group period is used?

**Comment by nothing-personal-gh** [comment]
@zeroin Thanks. Zoom into 15 minute period to reproduce this bug.

**Comment by martynasma** [comment]
Fixed in [5.2.27](https://github.com/amcharts/amcharts5/releases/tag/5.2.27).

## [5.2.27] - 2022-10-04

### Fixed
- `"average"` value of grouped data items was not being calculated properly.
- `WordCloud` was ignoring the `fill` value from label `templateField`.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
